Sources/TokamakCore/Views/ForEach.swift

```diff
@@ -12,6 +12,18 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+/// A structure that computes views on demand from an underlying collection of of identified data.
+///
+/// Available when `Data` conforms to `RandomAccessCollection`, `ID` conforms to `Hashable`, and `Content` conforms to `View`.
```

Comment:
<!--
  1 failure:  Line should be 10...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: Sources/TokamakCore/Views/ForEach.swift;
  Line: 17;
-->
- :no_entry_sign: Line should be 100 characters or less: currently 126 characters (`line_length`)